plugins/pastebins/mozilla.py

```diff
@@ -0,0 +1,43 @@
+import requests
+from requests import HTTPError, RequestException
+from cloudbot import hook
+from cloudbot.util.web import Pastebin, pastebins, ServiceHTTPError, ServiceError
+
+class Mozilla(Pastebin):
+    def __init__(self):
+        super().__init__()
+        self.url = "https://paste.mozilla.org/"
+
+    def paste(self, data, ext):
+        if isinstance(data, str):
+            encoded = data.encode()
+        else:
+            encoded = data
+
+        try:
+            if ext == 'txt':
+                lexer = '_text'
```

Comment:
nitpicking, but maybe we can set it to markdown ?

Reply:
Done! added support for markdown but fails back to text.